### ISSUE TITLE
Adding or replacing the upscaler

### ISSUE THREAD

**Issue by ThePeacemaker82** [divamgupta/diffusionbee-stable-diffusion-ui#305]
Wondering if it's possible to add a different upscaler, specifically Remacri. 
Or alternatively does anyone is willing to help guiding me on how to replace the original one with Remacri?

Thanks!

**Comment by barryanders** [comment]
I was about to say something to this effect. Someone did some nice comparisons over here for reference: https://www.reddit.com/r/StableDiffusion/comments/yr9xbk/extensive_research_of_upscalers_remacri_has_best/

Remacri seems to have an edge at least in the results I looked at over there.

**Comment by ThePeacemaker82** [comment]
yeah @barryanders, that's exactly how I discovered that upscaler too, never heard of it and been surprised by the results emerged on that stable diffusion thread. 
Hope it could be integrated in DiffusionBee!

**Comment by Gitterman69** [comment]
+1

**Comment by barryanders** [comment]
This showed up a couple weeks ago. It appears to be an upscaler from CompVis. I have no idea if it's usable as is or if they haven't released it fully yet, but seeing how it's from CompVis, I have high hopes that it will trump other upscalers.

https://huggingface.co/CompVis/ldm-super-resolution-4x-openimages

**Comment by LeaveNhA** [comment]
Is there any road-map or something actionable that we can research, study or implement on?

**Comment by xirtus** [comment]
I would love to see batch upscaling even upscaling directly of Gifs, and I'd LOVE to see Remacri. https://huggingface.co/utnah/LDSR/blob/main/4x_foolhardy_Remacri.pth

**Comment by xirtus** [comment]
is it possible to add the pth files or do we need to use param and .bin, noticed a great repository openmodeldb, some great options there in small form factor https://openmodeldb.info/models/4x-UltraSharp
